### PR TITLE
Project/background

### PR DESCRIPTION
**Pull request by AshiJoshi**
Add background to the project page

### CODE REVIEW

**Comment by AshiJoshi** [comment]
@tintinkung Oh I didn't realize that. Would you recommend not using it?

**Comment by tintinkung** [comment]
> Would you recommend not using it?

uh yea

File: index.html [hunk 1]
```diff
@@ -126,7 +127,7 @@ <h2>Demonarcy Monument</h2>
       <div class="slide-icon"></div>
     </div>
   </div>
-  <div class="text-center my-3">
+  <div class="text-center navbar-expand-lg navbar-light bg-light" style="padding:top 10px;">
     <a href="https://discord.gg/46Ed3UUJRD"><i class="footer_icon fab fa-discord"></i></a>
```
[the remark this answers is not in the document]
Comment (tintinkung) [reply index.html]: I resolved it by removing mdbootstrap sorry 🙇

**Comment by tintinkung** [comment]
### sorry i removed mdbootstrap making pr ready to merge.
gotta merge now thanks 😄

**Pull request merged**